src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/Library.kt

```diff
@@ -8,9 +8,6 @@ import org.springframework.data.annotation.Id
 import org.springframework.data.mongodb.core.mapping.Document
 
 @Document("libraries")
-@NoArgsConstructor
```

Comment:
These are extra annotations we don't need (if it was Java we may have but Kotlin is nice).